docs/releases/1.11.1.txt

```diff
@@ -52,3 +52,6 @@ Bugfixes
 
 * Corrected the stack level of unordered queryset pagination warnings
   (:ticket:`28109`).
+
+* Fixed an issue with nested ``__in`` subquery lookups when using ``to_field``
```

Comment:
Fixed a regression causing incorrect queries for __in subquery lookups when models use ForeignKey.to_field
(I think the commit message should also describe the problem solved rather than the implementation details.)